kDriveCore/Data/Api/Endpoint.swift

```diff
@@ -137,15 +143,21 @@ extension File: AbstractFile {}
 
 public extension Endpoint {
     private static var drive: Endpoint {
-        return Endpoint(hostKeypath: \.apiDriveHost, path: "/2/drive")
+        return Endpoint(hostKeypath: \.apiDriveHost, path: "/3/drive")
     }
 
     static var inAppReceipt: Endpoint {
         return Endpoint(path: "/invoicing/inapp/apple/link_receipt")
     }
 
+    // MARK: V2
+
+    private static var driveV2: Endpoint {
```

Comment:
This is because we originally still had V1 endpoints. I think this is still better to use V2 because the api does endpoint versioning so some endpoints aren't legacy they have simply never needed to move to V3.

Reply:
Okay, no legacy then. But what can we do to be consistent within the naming within the file ?
Looking at the code, just this one is named that way, and it could be confusing. (given the non suffixed one is the V3). 

Can we just rename `drive` `driveV3` maybe ?